browser/extensions/brave_component_loader.cc

```diff
@@ -97,15 +97,6 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
   }
 #endif
 
-  if (!command_line.HasSwitch(switches::kDisableWebTorrentExtension) &&
-      (!profile_prefs_->FindPreference(kWebTorrentEnabled) ||
-      profile_prefs_->GetBoolean(kWebTorrentEnabled))) {
-    base::FilePath brave_webtorrent_path(FILE_PATH_LITERAL(""));
-    brave_webtorrent_path =
-      brave_webtorrent_path.Append(FILE_PATH_LITERAL("brave_webtorrent"));
-    Add(IDR_BRAVE_WEBTORRENT, brave_webtorrent_path);
-  }
```

Comment:
This is moved into `AddWebTorrentExtension` which will only be called from the network throttle on the first magnet or Torrent load.